### DISCUSSION TITLE
This quite normal beatmap has a big lagspike

### DISCUSSION THREAD

**Discussion by Norceosu** [ppy/osu#24441] [Q&A] [answered]
https://osu.ppy.sh/beatmapsets/1219178#osu/2536511
00:12:727 - Is around here where the frame rate drops a lot (from 360 to 40 fps)

[updater.log](https://github.com/ppy/osu/files/12231115/updater.log)
[database.log](https://github.com/ppy/osu/files/12231116/database.log)
[input.log](https://github.com/ppy/osu/files/12231117/input.log)
[network.log](https://github.com/ppy/osu/files/12231118/network.log)
[performance.log](https://github.com/ppy/osu/files/12231119/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12231120/runtime.log)

**Comment by peppy** [comment, marked as answer]
This is because the map is using silly mapping techniques. The sliders in this map have thousands of control points.

I don't know if/when we will support this, but it's a low priority either way.
